data/templates/dhcp-server/dhcpdv6.conf.j2

```diff
@@ -11,6 +11,13 @@ option dhcp6.preference {{ preference }};
 {% if global_parameters.name_server is vyos_defined %}
 option dhcp6.name-servers {{ global_parameters.name_server | join(', ') }};
 {% endif %}
+{% if global_parameters.cisco_voip.length_width is vyos_defined and global_parameters.cisco_voip.width is vyos_defined  %}
+option space ciscovoip code width {{ global_parameters.cisco_voip.width }} length width {{ global_parameters.cisco_voip.length_width }};
+option vsio.ciscovoip code 9 = encapsulate ciscovoip;
+{% endif %}
+{% if global_parameters.cisco_voip.tftp_servers is vyos_defined %}
```

Comment:
The option statement can always be defined globally, it' sonly a mnemonic for dhcpd on how to send data if it's defined, thus we should not add additional complexity and drop the if.